lib/business_time/config.rb

```diff
@@ -7,16 +7,34 @@ module BusinessTime
   # manually, or with a yaml file and the load method.
   class Config
     DEFAULT_CONFIG = {
-      holidays:              [],
-      beginning_of_workday:  '9:00 am',
-      end_of_workday:        '5:00 pm',
+      holidays:              SortedSet.new,
+      beginning_of_workday:  ParsedTime.parse('9:00 am'),
+      end_of_workday:        ParsedTime.parse('5:00 pm'),
       work_week:             %w(mon tue wed thu fri),
       work_hours:            {},
       work_hours_total:      {},
       _weekdays:             nil,
     }
 
     class << self
+      def beginning_of_workday=(time)
+        config[:beginning_of_workday] = ParsedTime.parse(time)
+      end
+
+      def end_of_workday=(time)
+        config[:end_of_workday] = ParsedTime.parse(time)
+      end
+
+      def work_hours=(work_hours)
+        config[:work_hours] = {}
+        work_hours.each do |day, hours|
```

Comment:
this is all style and minor, but maybe you could use `each_with_object` here to set the `:work_hours` key in the `config` hash at the same time you build it, saving maybe two lines of code for the initialization and assignment steps.